aws_lambda_powertools/utilities/advanced_parser/envelopes/sqs.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from typing import Any, Dict, List
+
+from pydantic import BaseModel, ValidationError
+
+from aws_lambda_powertools.utilities.advanced_parser.envelopes.base import BaseEnvelope
+from aws_lambda_powertools.utilities.advanced_parser.schemas import SqsSchema
+
+logger = logging.getLogger(__name__)
+
+
+class SqsEnvelope(BaseEnvelope):
```

Comment:
docs: comment or docstring on the return if possible

Reply:
the type hint is good here but i'll add a comment.